src/styles/_header.scss

```diff
@@ -0,0 +1,78 @@
+.header {
+  height: 765px;
+  background-image: url("../images/header.jpg");
+
+  @include cover-image;
+  position: relative;
+  font-weight: 500;
+
+  &__nav {
+    position: absolute;
+    bottom: 25px;
+    left: 0;
+    right: 0;
+  }
+
+  &__top {
+    top: 35px;
+    left: calc(50% - #{$logo-width/2});
+
+    display: flex;
+    position: absolute;
+
+    color: $main-color;
+  }
+
+  &__phone {
+    margin: 0 82px;
+  }
+
+  &__phone,
+  &__action {
+    top: 47px;
+    position: relative;
+
+  }
+
+  &__link {
+    text-decoration: none;
+    border-bottom: 1px dashed;
+    text-transform: uppercase;
+    
+    color: $main-color;
+    transition: color $hover-duration;
+
+    &:hover {
+      color: $active-color;
+    }
+  }
+}
+
+@media screen and (max-width: $small-screen-breakpoint) {
+  .header {
+    &__top {
+      flex-direction: column;
+    }
+
+    &__phone {
+      margin: -30px 0 10px;
+    }
+  }
+}
+
+// @media screen and (max-width: 320px) {
```

Comment:
it's better not to leave unnecessary comments

Reply:
done)